concordia/models.py

```diff
@@ -396,6 +396,7 @@ class SiteReport(models.Model):
     campaign = models.ForeignKey(
         Campaign, on_delete=models.DO_NOTHING, blank=True, null=True
     )
+    topic = models.ForeignKey(Topic, on_delete=models.DO_NOTHING, blank=True, null=True)
```

Comment:
Do we want `DO_NOTHING` or `SET_NULL`? The former would appear to prevent a topic from ever being deleted without manually updating the database first.